.gitattributes

```diff
@@ -0,0 +1,4 @@
+# Configure git to use `ansible-vault view` for diff'ing these files with:
+# git config --global diff.ansible-vault.textconv "ansible-vault view"
+etc/kayobe/secrets.yml diff=ansible-vault merge=binary
```

Comment:
How are you setting the vault password? Could we pick up  `KAYOBE_VAULT_PASSWORD` from the system env?

Reply:
Sorry - clearly didn't read your commit message - but question still stands.

Reply:
Vault password can be provided via `ANSIBLE_VAULT_PASSWORD_FILE` env var so theoretically that could be set at the same time as `KAYOBE_VAULT_PASSWORD`. In a local dev environment it's preferable to keep the vault password inside the repo so the `~/.ansible.cfg` approach works with multiple vaults.

Reply:
I was hoping it could be more automatic, but not to worry, something for https://github.com/markgoddard/beokay maybe